nixos/modules/services/databases/openldap.nix

```diff
@@ -242,6 +242,12 @@ in
       }
     ];
 
+    warnings = [ ''
+      The openldap module is unmaintained and staged for removal.
+      If you use it, please fix it and remove this message.
+      See https://github.com/NixOS/nixpkgs/issues/86305 for details.
```

Comment:
This is not true. On my server I run openldap using nixpkgs-unstable and regulary update it.
If anything breaks I will fix it.

Reply:
Well, the module doesn't have a maintainers attribute currently, so I can't know ;-)

I'd personally think a simple VM test is more sustainable having you fixing breakages after-the fact.

For example, I planned refactoring some of the unit parameters, but couldn't verify things still work, as there was no passing test.

Reply:
Add me as a maintainer than.

Reply:
Done, thanks!